apex-test-kit/main/classes/ATKTest.cls

```diff
@@ -17,6 +17,22 @@
 @IsTest
 public with sharing class ATKTest {
 
+    @IsTest
+    static void test_SaveRecord() {
+        Test.startTest();
+        ATK.SaveResult result = ATK.prepare(Account.getSObjectType(), 1)
+            .field(Account.Name).index('Account #{0}')
+            .save();
```

Comment:
Unfortunately, the test method should avoid perform any actual save into the database, because we have different orgs with different required field.

Reply:
You are right. But it is the easiest way to test that method for now. If the package were managed this wouldn't be an issue. 🤔 And I can see why it is not. I'll think about how to test this without making gimmicks around the code.